.pre-commit-config.yaml

```diff
@@ -29,3 +29,11 @@ repos:
       - id: flake8
         additional_dependencies:
           - flake8-docstrings~=1.6.0
+
+  - repo: https://github.com/psf/black
+    rev: 22.6.0
+    hooks:
+      - id: black
+        language_version: "3.10"
+        args:
+          - --line-length=119
```

Comment:
I have tested it on EndeavourOS and it works great. However, if the `flake8` tests pass, formatting isn't needed. Though it is still useful when the tests fail.

Reply:
Does flake8 check for everything that black does? [Here](https://flake8.pycqa.org/en/latest/user/error-codes.html) is the list of default codes that we get with installing flake8, as well as #2 to come.

Reply:
I am not sure, but `black` is useful anyway, so we can just keep it.